src/commands/suggestions/suggestionsUpdate.ts

```diff
@@ -0,0 +1,60 @@
+import { Message, MessageEmbed } from 'discord.js';
+import { CommandoClient, CommandoMessage } from 'discord.js-commando';
+
+import { AdminCommand } from '../../utils/commands';
+import {
+  suggestionStatesReadable,
+  getAvailableStatesString,
+  updateSuggestionState,
+  SuggestionState
+} from '../../components/suggestions';
+import { EMBED_COLOUR } from '../../utils/embeds';
+import { parseStateArg, validateState, validateIDs } from './utils';
+
+class SuggestionsUpdateCommand extends AdminCommand {
+  constructor(client: CommandoClient) {
+    super(client, {
+      name: 'suggestions-update',
+      aliases: ['suggestion-update'],
+      group: 'suggestions',
+      memberName: 'update',
+      args: [
+        {
+          key: 'state',
+          prompt: `enter one of ${getAvailableStatesString()}.`,
+          type: 'string',
+          validate: validateState,
+          parse: parseStateArg
+        },
+        {
+          key: 'ids',
+          prompt: `enter suggestion IDs seperated by spaces.`,
+          type: 'string',
+          validate: validateIDs
+        }
+      ],
+      description: 'Updates a list of suggestion states.',
+      examples: [
+        `${client.commandPrefix}suggestions-update actionable 1 2 3`,
+        `${client.commandPrefix}suggestion-update created 99`
+      ]
+    });
+  }
+
+  async onRun(message: CommandoMessage, args: { state: string; ids: string }): Promise<Message> {
+    const { state, ids } = args;
+    const suggestionIds = ids.split(' ').map((a) => Number(a));
+    const suggestionState = state as SuggestionState;
+
+    // Update states
+    await updateSuggestionState(suggestionIds, suggestionState);
```

Comment:
Now that you're using this for general updates, please update/remove the comment in the function itself in `components/suggestions.ts` to reflect that it does more than just update created to pending.

Reply:
Also, are we just letting the ids that don't exist just fall through? We aren't planning on providing any warnings/errors for that?

Reply:
Mistyping a number (that exists) and messing up data is probs worse than changing ids that don't exist (does nothing to DB). Like there are no warnings if I turn some actually rejected suggestions back into created.
If we shouldn't have data mess-ups, we shouldn't have ids that DNE, so mods gotta be a bit more careful.

Reply:
hmm if mistyping is easier than typing a num that doesnt exist...
@gzcharleszhang how might a ``confirm`` similar to how a ``cancel`` cancels work? (if its too complicated then lol just have good mods)

Reply:
let's just leave it like this for now